app/assets/javascripts/front/realization.js

```diff
@@ -0,0 +1,253 @@
+    this.remindForm.addEventListener('ajax:error', (event, xhr, status, error) => this.formError())
+
+    this.target = 'intro'
+    
+    this.introBtn.addEventListener('click', () => this.nextScreen(this.introBtn))
+    this.readyBtn.addEventListener('click', () => this.nextScreen(this.readyBtn))
+    this.readyRemindBtn.addEventListener('click', () => this.nextScreen(this.readyRemindBtn))
+    this.pausedBtn.addEventListener('click', () => this.nextScreen(this.pausedBtn))
+    this.pausedFinishBtn.addEventListener('click', () => this.nextScreen(this.pausedFinishBtn))
+    this.remindBackBtn.addEventListener('click', () => this.nextScreen(this.remindBackBtn))
+    this.surveyNextBtn.addEventListener('click', () => this.passFeel())
+    this.thanksBtn.addEventListener('click', () => this.share())
+    
+    for (let i = 0; i < this.surveyBtns.length; i++) {
+      const surveyBtn = this.surveyBtns[i]
+      surveyBtn.addEventListener('click', () => surveyBtn.classList.toggle('button--active'))
+    }
+    
+    if (typeof zenscroll !== 'undefined') {
+      this._scrollTo(this.target)
+    }
+
+  }
+
+  _scrollTo(target) {
+    let el = document.getElementById(target)
+    zenscroll.toY(zenscroll.getTopOf(el) - 30)
+  }
+
+  nextScreen(btn) {
+    switch (btn) {
+    case this.introBtn:
+      this.target = 'ready'
+      this.container.dataset.screen = this.target
+      break
+    case this.readyBtn:
+      this.target = 'video'
+      this.container.dataset.screen = this.target
+      if ($(this.desktopVideo).is(':visible')) {
+        // this.setupDesktopPlayer()
+        jwplayer('botr_1oDJAjaD_8SfP7aMx_div').on('ready', this.loadJWPlayer('botr_1oDJAjaD_8SfP7aMx_div'))
+        jwplayer('botr_1oDJAjaD_8SfP7aMx_div').play()
+      } else {
+        // this.setupMobilePlayer()
+        jwplayer('botr_8FivThod_NaHiexhY_div').on('ready', this.loadJWPlayer('botr_8FivThod_NaHiexhY_div'))
+        jwplayer('botr_8FivThod_NaHiexhY_div').play()
+      }
+      break
+    case this.readyRemindBtn:
+      this.target = 'remind'
+      this.container.dataset.screen = this.target
+      break
+    case this.pausedBtn:
+      this.target = 'video'
+      this.container.dataset.screen = this.target
+      break
+    case this.pausedFinishBtn:
+      this.target = 'survey'
+      this.container.dataset.screen = this.target
+      break
+    case this.remindBackBtn:
+      this.target = 'ready'
+      this.container.dataset.screen = this.target
+      break
+    default:
+      break
+    }
+
+    this._scrollTo(this.target)
+
+  }
+
+  setupDesktopPlayer() {
+    let jwConfig = {
+      aspectratio: '16:10',
+      autostart: false,
+      cast: {
+        appid: '00000000'
+      },
+      controls: true,
+      displaydescription: true,
+      displaytitle: true,
+      flashplayer: '//ssl.p.jwpcdn.com/player/v/8.17.7/jwplayer.flash.swf',
+      height: 360,
+      key: 'tlbC+HgyRBgC8seKreQwmghcO2P9nQ4y+l4ZeKQEfzpgOSzuT9f5+N0odL4=',
```

Comment:
Is this an API key? If so we should pass it by an environment variable instead of writing it in directly.